### PR TITLE
Pxweb2 87 ingress

### PR DESCRIPTION
**Pull request by KentMossback**
Added ingress component

### CODE REVIEW

File: libs/pxweb2-ui/src/lib/components/Typography/Ingress/Ingress.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import styles from './Ingress.module.scss';
+
+interface IngressProps {
+  children: React.ReactNode;  
+}
+
+export const Ingress: React.FC<IngressProps> = ({ children }) => {
+  return (
+    <div className={styles.ingress}>
```
Comment (SjurSutterudSagen) [inline libs/pxweb2-ui/src/lib/components/Typography/Ingress/Ingress.tsx:10]: Should we return something other than a div here? 'p' maybe?

Comment (michaelpande) [reply libs/pxweb2-ui/src/lib/components/Typography/Ingress/Ingress.tsx:10]: A <div> is a block. Which means it does not work as text. If you place a ingress inside a <p> tag with another piece of text, it would not be on the same line. The div block would not be on the same line as the rest of the text.

Consider using a span instead. A <p> tag is also a bit limiting, as it could not be used inside another <p>-tag. As that is semantically not valid. 

Alternatively the .ingress style in css could use display: inline, then the div would work as a span.

File: libs/pxweb2-ui/src/lib/components/Typography/Ingress/Ingress.module.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
+.ingress {
+    font-family: PxWeb-font-400;
+    font-size: 1.25rem;
+    //font-style: normal; // ??
```
Comment (michaelpande) [inline libs/pxweb2-ui/src/lib/components/Typography/Ingress/Ingress.module.scss:4]: Consider removing the unused code

**Review by michaelpande — APPROVED** [review]

File: apps/pxweb2/public/theme/variables.css
Comment (MikaelNordberg) [inline apps/pxweb2/public/theme/variables.css:1]: Should not be part of this PR

**Review by MikaelNordberg — COMMENTED** [review]
Looks good except for changes in variables.css

**Review by MikaelNordberg — DISMISSED** [review]
Looks good

**Review by MikaelNordberg — APPROVED** [review]
Looks good

**Review by PerIngeVaaje — APPROVED** [review]
Looks OK
